### PR TITLE
K8S-1127 fixes default port for auth service

### PR DESCRIPTION
**Pull request by xgerman**

### CODE REVIEW

**Comment by tonytan4ever** [comment]
Release notes needed.

**Comment by tonytan4ever** [comment]
Needs rebase and gate again.

**Comment by xgerman** [comment]
Bump

File: playbooks/files/rax-maas/plugins/managed_k8s_services_local_check.py [hunk 1]
```diff
@@ -68,11 +68,11 @@ def check(auth_ref, args):
         milliseconds = (dt.microseconds + dt.seconds * 10 ** 6) / 10 ** 3
 
     status_ok(m_name=service_name)
-    metric_bool('mk8s_{service}_local_status'.format(service=args.service),
+    metric_bool('managed_k8s_{service}_local_status'.format(service=args.service),
```
Comment (tonytan4ever) [inline playbooks/files/rax-maas/plugins/managed_k8s_services_local_check.py]: This line expands to long so pep8 check failed:

/var/lib/jenkins/workspace/PR_rpc-maas-master-container-pep8-tox-test/rpc-maas/playbooks/files/rax-maas/plugins/managed_k8s_services_local_check.py:71:80: E501 line too long (82 > 79 characters)

Need to wrap line around

**Comment by tonytan4ever** [comment]
recheck_xenial_ceph_deploy

**Comment by tonytan4ever** [comment]
recheck_xenial_ceph_deploy

**Comment by tonytan4ever** [comment]
recheck_trusty_mitaka_deploy 
recheck_xenial_pike_deploy

**Pull request merged**